v3.2/glfw/build.go

```diff
@@ -4,7 +4,7 @@ package glfw
 // Windows Build Tags
 // ----------------
 // GLFW Options:
-#cgo windows CFLAGS: -D_GLFW_WIN32
+#cgo windows CFLAGS: -D_GLFW_WIN32 -Iglfw/deps/mingw
```

Comment:
Given this is Windows, is it preferable to use `/` as path separator, or is it better to use backslash, making this:

```
-Iglfw\deps\mingw
```

Reference: https://github.com/golang/go/blob/fcfd91858b7b57de0577e98703390f604971eaf4/src/os/path_windows.go#L8